Runtime/Visual/CameraColorOverlay.cs

```diff
@@ -234,7 +234,7 @@ protected virtual void CancelBlinkRoutine()
         /// <param name="cam">The <see cref="Camera"/> to apply onto.</param>
         protected virtual void PostRender(Camera cam)
         {
-            if (ValidCameras == null || !ValidCameras.cameras.Contains(cam))
+            if (CameraValidity == null || !CameraValidity.Accepts(cam))
```

Comment:
```suggestion
            if (!CameraValidity.Accepts(cam))
```
`Accepts` is an extension method that already handles the rule being `null`.

While this change would make this rule usage consistent with all the other it does change the logic here, though:
With the suggested change a `null` rule would allow **any** camera, while currently we bail out treating it as no camera being valid instead.

What shall we do? 🤔

Reply:
Whilst before it would never apply to all scene cameras if it was null, my gut is telling me that if you're using a rule and you don't apply a restriction then no restrictions should apply.

So I vote for removing the `== null` check